evennia/contrib/base_systems/character_creator/README.md

```diff
@@ -0,0 +1,87 @@
+# Character Creator contrib
+by InspectorCaracal
+
+This contrib is designed to be used in MULTISESSION_MODE = 2 or higher, where characters are not automatically created to match the account. To use this with lower modes, you'll need to implement your own solution for preventing the built-in automatic character creation.
+
+## Installation
+
+In your game folder `commands/default_cmdsets.py`, import and add `ContribCmdCharCreate` to your `AccountCmdSet`.
+
+Example:
+```py
+from evennia.contrib.base_systems.character_creator.character_creator import ContribCmdCharCreate
+
+class AccountCmdSet(default_cmds.AccountCmdSet):
+    
+    def at_cmdset_creation(self):
+        super().at_cmdset_creation()
+        self.add(ContribCmdCharCreate)
+```
+
+In your game folder `typeclasses/accounts.py`, import and inherit from `ContribChargenAccount` on your Account class.
+
+(Alternatively, you can copy the `at_look` method directly into your own class.)
+
+Example:
+```py
+from evennia.contrib.base_systems.character_creator.character_creator import ContribChargenAccount
```

Comment:
I'm not sure this belongs in `contribs/base_systems`. I can see the argument for this be a sort of 'base system' for character generation, but on the other hand a character generation system is very much a RPG-staple, so wouldn't `contribs/rpg` be more suitable?